docs/symbols.bzl

```diff
@@ -13,8 +13,7 @@ load(
     _rust_bindgen_repositories = "rust_bindgen_repositories",
 )
 load(
-    "@rules_rust//cargo:defs.bzl",
```

Comment:
This seems like a bad merge conflict resolution. Can you restore this and the other deleted things in this file?

Reply:
Redid the merge, looks better now, I think. Please take another look.